concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+Morse-1996-375-209	209	tu mano	thy hand			
+Morse-1996-375-210	210	su mano	his hand			
+Morse-1996-375-211	211	nuestras manos	our hands			
+Morse-1996-375-212	212	vuestras manos	your hands			
+Morse-1996-375-213	213	sus manos de ellos	their hands			
+Morse-1996-375-214	214	mi arco	my bow			
+Morse-1996-375-215	215	tu arco	thy bow			
+Morse-1996-375-216	216	su arco – de él	his bow			
+Morse-1996-375-217	217	nuestro arco	our bow			
+Morse-1996-375-218	218	vuestro arco	your bow			
+Morse-1996-375-219	219	su arco – de ellos	their bow			
+Morse-1996-375-220	220	grande	big	*	1202	BIG
+Morse-1996-375-221	221	pequeño	small	*	1246	SMALL
+Morse-1996-375-222	222	frío	cold	*	1287	COLD
+Morse-1996-375-223	223	caliente	hot	*	1286	HOT
+Morse-1996-375-224	224	bueno	good	*	1035	GOOD
+Morse-1996-375-225	225	malo	bad	*	1292	BAD
+Morse-1996-375-226	226	blanco	white	*	1335	WHITE
+Morse-1996-375-227	227	negro	black	*	163	BLACK
+Morse-1996-375-228	228	vaya, ve	go!			
+Morse-1996-375-229	229	ven	come!			
+Morse-1996-375-230	230	come	eat!			
+Morse-1996-375-231	231	bebe!	drink!			
+Morse-1996-375-232	232	duerme!	sleep!			
+Morse-1996-375-233	233	corona	crown of head		1943	CROWN (OF HEAD)
+Morse-1996-375-234	234	dientes delanteros incisivos	front teeth		442	FRONT TOOTH (INCISOR)
+Morse-1996-375-235	235	punta de la lengua	tip tongue		443	TIP (OF TONGUE)
+Morse-1996-375-236	236	pelo largo	long hair		444	LONG HAIR
+Morse-1996-375-237	237	toda la garganta y cuello	around the neck			
+Morse-1996-375-238	238	nuez de la garganta	Adam’s apple		802	ADAM'S APPLE
+Morse-1996-375-239	239	espalda, parte superior	upper back		445	UPPER BACK
+Morse-1996-375-240	240	antebrazo	lower arm		446	LOWER ARM
+Morse-1996-375-241	241	muñeca	wrist		799	WRIST
+Morse-1996-375-242	242	la pierna inferior	lower leg		447	LOWER LEG
+Morse-1996-375-243	243	el pelo del cuerpo	body hair		189	HAIR (BODY)
+Morse-1996-375-244	244	estómago	stomach		819	STOMACH
+Morse-1996-375-245	245	intestinos	intestines	*	1991	INTESTINES
+Morse-1996-375-246	246	viejita	old woman		907	OLD WOMAN
+Morse-1996-375-247	247	nubes quietas	clouds at rest	*	1489	CLOUD
+Morse-1996-375-248	248	nubes de tormenta	storm clouds		448	STORM CLOUDS
+Morse-1996-375-249	249	chagra	corn field		449	CORN FIELD
+Morse-1996-375-250	250	quebrada	stream		947	STREAM
+Morse-1996-375-251	251	piedrecillas	pebbles		961	PEBBLES
+Morse-1996-375-252	252	rocas	huge rocks			
+Morse-1996-375-253	253	trocha	path		2252	PATH
+Morse-1996-375-254	254	persianas	venetian blinds			
+Morse-1996-375-255	255	tambo	shelter			
+Morse-1996-375-256	256	este	this		1214	THIS
+Morse-1996-375-257	257	aquel	that		78	THAT
+Morse-1996-375-258	258	quien	who		1235	WHO
+Morse-1996-375-259	259	que	what		1236	WHAT
+Morse-1996-375-260	260	no	not		1240	NOT
+Morse-1996-375-261	261	todos	all		98	ALL
+Morse-1996-375-262	262	muchos	many		1198	MANY
+Morse-1996-375-263	263	largo	long		1203	LONG
+Morse-1996-375-264	264	corteza	bark		1204	BARK
+Morse-1996-375-265	265	carne	flesh		2615	FLESH OR MEAT
```

Comment:
As far as I can tell, 'carne' in Spanish potentially refers to both 'flesh' and 'meat' but if the given English gloss specifies just one of them (and it's the source language too), could it be the case that the meaning being elicited was narrowed down to just one of them?

Reply:
leave flesh or meat, I'd say, as we never know what they elicited.